### DISCUSSION TITLE
different table of contents title for each chapter in quarto html book?

### DISCUSSION THREAD

**Discussion by ElineVG** [quarto-dev/quarto-cli#3929] [Q&A] [answered]
Is it possible to change the toc-title in a quarto book (html output) depending on the chapter that is currently visible on the screen? 

For example, instead of 'Table of contents' it would say 'Chapter 1' or 'Chapter 2' depending on the chapter currently on the screen?
How can I achieve that?

Thank you very much in advance!

**Comment by mcanouil** [comment, marked as answer]
I am not sure you can ...

Did you try to add `toc-title: Contents` yaml entry as part of your different chapters yaml headers? (I don't think it will work, but worth the try).

**Reply by ElineVG** [reply]
Thank you, that works!

[The solution I found before seeing your answer was to set the overall toc-title, give it an id, and then change the toc-title per chapter with Javascript; but your solution is cleaner.]

**Reply by mcanouil** [reply]
I am surprised it worked ^^
I assumed as for some other options, each new entry would have overwrite and took precedence over all those before.
